website/settings/local-travis.py

```diff
@@ -87,3 +87,6 @@
 NEW_AND_NOTEWORTHY_LINKS_NODE = 'helloo'
 POPULAR_LINKS_NODE = 'hiyah'
 POPULAR_LINKS_REGISTRATIONS = 'woooo'
+
+EZID_USERNAME = 'testfortravisnotreal'
```

Comment:
Without this set in travis, two tests that I have not added or modified  -- [this test here and the one below it](https://github.com/CenterForOpenScience/osf.io/blob/develop/tests/test_identifiers.py#L129) fail because of a check that I did add on `request_identifiers_from_ezid` - namely that if `if settings.EZID_USERNAME and settings.EZID_PASSWORD` aren't set, then nothing in that method is run and no errors are thrown, so that a preprint can still be created just without any identifiers. The two tests linked above are testing methods that are called from within `request_identifiers_from_ezid` that are already mocked using httpretty in the tests themselves.  So this is just to make two tests that were already there pass with an extra check on `request_identifiers_from_ezid` that wasn't really needed before because identifiers weren't automatically created.